### ISSUE TITLE
Problem accepting multiple Accept attributes

### ISSUE THREAD

**Issue by bbarker** [jjn1056/Catalyst-ActionRole-MatchRequestAccepts#1]
Hi,

First, thanks for the module, it really made working with conneg quite easy. I have one possible issue:

My understanding of the docs was that a Catalyst action should be able to accept multiple Accept attributes ("Please note that if you specify multiple Accept attributes on a single action, those will be matched via an OR condition and not an AND condition"), e.g.:

``` perl
sub for_html : Path('/art') Accept('plain/html') Accept('text/html') Args(1) {
#...
```

This didn't seem to work; instead I had to forward to another:

``` perl
sub for_text_html : Path('/art') Accept('text/html') Args(1) {
    my ( $self, $c, $id_in ) = @_;
    $c->forward('for_plain_html');
}

sub for_plain_html : Path('/art') Accept('plain/html') Args(1) {
#...
```

**Comment by tsibley** [comment]
I reported this same issue as https://rt.cpan.org/Ticket/Display.html?id=107786 and submitted a PR to fix it: https://github.com/jjn1056/Catalyst-ActionRole-MatchRequestAccepts/pull/2

**Comment by tsibley** [comment]
Fixed now with #2 merged.

**Comment by bbarker** [comment]
Thanks Guys
